earth_enterprise/tutorial/FusionTutorial/download_tutorial.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
It occurs to me that we should have some instructions about this somewhere.  Maybe we could add a note to the wiki where it talks about installing the tutorial files: https://github.com/google/earthenterprise/wiki/Install-Fusion-or-Earth-Server.